### ISSUE TITLE
Clarification on PepMLM's Masking Strategy

### ISSUE THREAD

**Issue by charlotte-ris-MIT** [programmablebio/pepmlm#1]
I have a question about the masking strategy used by PepMLM, as discussed in the paper. Specifically, the paper mentions positioning the peptide binder sequence at the terminus of target protein sequences. I'm curious if this implies that the binder is always located at the N-terminus. Also, I'm interested in knowing if there are any strategies for putting the peptide binder closer to specific hotspots.

**Comment by Leo-T-Zang** [comment]
You could insert the binder near the hotspot region and fine-tune the model. We haven’t tried this before, as we think it would largely deviate from pretraining and might not be optimal.
